qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -319,5 +318,39 @@ private static void createTestSamFile( ) {
 					Level.WARNING, "IOException caught whilst attempting to write to SAM test file: " + input.getAbsolutePath(), e);
 		}  
 	}
-	
+		 
+	@Test
+	public void maxLengthTest() throws IllegalArgumentException {	
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause